processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
QueryMetrics is documented to be broken often, so there are no compatibility concerns about this interface.

> You mean just many users want to filter metrics by data source alone without intervals?

Yes.